app/_layout.tsx

```
@@ -1,105 +1,112 @@
-
-initSentry()
+import { useColorScheme } from "@/lib/hooks/useColorScheme";
+import { useFonts } from "expo-font";
+import { UserProvider } from "@/lib/context/global/user.context";
+import { useEffect } from "react";
+import { Stack } from "expo-router";
+import { StatusBar } from "expo-status-bar";
+import FlashMessage from "react-native-flash-message";
+import InternetProvider from "@/lib/context/global/internet-provider";
+import AnimatedSplashScreen from "@/lib/ui/useable-components/splash/AnimatedSplashScreen";
+import { SoundProvider } from "@/lib/context/global/sound.context";
```

ESLint

[error] 'SoundProvider' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)